charts/seed-machines/charts/packet-machineclass/values.yaml

```diff
@@ -0,0 +1,14 @@
+machineClasses:
+- name: class-1
+  projectID: abcd-1234-fff
+  OS: alpine_3
+  facility: 
+  - EWR1
+  description: An optional description for machines created by that class.
+  machineType: t1.small
+  secret:
+    cloudConfig: abc
+    packetAPIKey: "abd5678"
+  tags:
+  - my-nodes
```

Comment:
```suggestion
    my-nodes
```

Reply:
Shoot, it isn't. It actually is a list of tags, i.e. `[]string`, and not a key/value map. I need to fix this elsewhere as well.

Reply:
Do you mind explaining what that error message you showed is?

Reply:
This is resolved now.